homeassistant/components/synology_dsm/const.py

```diff
@@ -227,15 +228,15 @@
     f"{SynoStorage.API_KEY}:disk_smart_status": {
```

Comment:
`f"{SynoStorage.API_KEY}:disk_smart_status"` and `f"{SynoStorage.API_KEY}:disk_status"` are sensor not binary_sensor

Reply:
ok, so... what do we need to go on?  had approval 2 days ago, until you came along and things started to become difficulty ;-) Are you saying these can't have the device_class set? which was the point for the beginning of course...

Reply:
I understand that is difficult for you, I should review everything one time, and things that can be nice for people who don't know the code, but is not in real.

I'm saying that this should be binary_sensor not sensor, so there is not only the `const.py` to modify.

Reply:
ok, but that's another issue altogether? shouldn't we first do this, and after wards see if the team agrees on changing from binary to sensor (which I do agree on btw)
otherwise this PR is diverted too much I fear

Reply:
See 1 line up